packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -28,6 +28,13 @@ export const styles = css`
         align-items: center;
     }
 
+    :host(.nested-parent-row-first-cell) .cell-view {
+        padding-left: calc(
```

Comment:
I'm not clear on how all these paddings work. 

1. Under various circumstances, there are left paddings on the cell, the cell view, the expand button, and there's a left margin on the cell container. Also, there's a front row spacer. I don't think any of these are overlaying each other, but it also doesn't seem like these would make sense all in series horizontally.
2. It seems like `controlHeight` is the width of the indentation (of a child row under its parent), but I don't understand why `1` is being subtracted from  `--ni-private-table-cell-nesting-level` here and in the row styles.

Naively, I would expect the left padding/margin to be on just one of the elements (or maybe no padding, but the the row front spacer's width would account for it). There's likely a reason for doing things this way, but it's not obvious to me. I just wanted to bring this up in case you can think of a way to simplify it or maybe just give some insight in a comment somewhere.

Reply:
@m-akinc and I talked through this off-line. In summary, we think it works as needed and is clear enough particularly with the caveat that I was able to address his 2nd concern (I had dumb math in my CSS).

If others have similar concerns we can discuss the best way to resolve them, be it an alternative strategy or some form of inline docs.